vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -348,7 +350,7 @@ def set_relationship_on_hosts(hosts)
       #
 
       def normalize_blank_property_num(property)
-        property.try(:to_i)
+        property.try(:to_i) || 0
```

Comment:
Right.... so `property.try(:to_i) || 0` == `property.to_i`

(In which case, maybe `normalize_blank_property_num` should just go away?)